activemq-spring/pom.xml

```diff
@@ -87,6 +87,11 @@
       <artifactId>org.linkedin.zookeeper-impl</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.ant</groupId>
+      <artifactId>ant</artifactId>
+      <scope>provided</scope>
+    </dependency>
```

Comment:
Is this actually needed? It was only a transitive dependency before, sio unless the module has its own Ant usage it seems like it should still be transitive. 

The dependencyManagement will still control the version without adding this, and the scope be based on the scope of whatever pulls it in transitively.

Reply:
It was what my mean in my previous comment, but I think I was not clear.